code/modules/client/client_procs.dm

```diff
@@ -933,3 +933,45 @@ GLOBAL_LIST_INIT(whitelisted_client_procs, list(
 		winset(src, "mapwindow.map", "right-click=false")
 		winset(src, "default.Shift", "is-disabled=true")
 		winset(src, "default.ShiftUp", "is-disabled=true")
+
+GLOBAL_LIST_INIT(community_awards, get_community_awards())
+
+/client
+	var/list/community_awards = list()
```

Comment:
where is this used?

Reply:
![image](https://github.com/user-attachments/assets/4fcdcf9f-6af7-42f3-a5d2-efeeb5bf412d)
Right below your comment